install.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+# MAKE DIRECTORY SO THAT YOUR HOME DIRECTORY DOESN'T GET DIRTY
+
+mkdir -p junk-drawer
+cd junk-drawer
+
+# Install git and wget
+
+pacman -S wget git --needed
```

Comment:
This will fail on distributions that aren't Arch-based. Could fall back to `curl`, and if that also fails, quit early and prompt the user to install.

Reply:
lemme add a thingy that'll ask for their distro

Reply:
> lemme add a thingy that'll ask for their distro

done